test/test_sort_and_select.py

```diff
@@ -438,6 +438,11 @@ def compare(t, k, dim, dir):
         compare(t, 2000, 1, True)
         compare(t, 2000, 1, False)
 
+    def test_topk_quantized_scalar_input(self):
+        # This used to segfault
```

Comment:
same comment as other PR please put the issue number here to make tracking the segfault easier if it reapears.